src/utils/buffer.ts

```diff
@@ -54,8 +54,8 @@ export function signedIntToBytes(value: number, bytes: number): Buffer {
     view.setInt32(0, value, false);
   } else if (bytes === 8) {
     // In case of 8 bytes I need to handle the int with a Long lib
-    let long = Long.fromNumber(value, false);
-    arr = long.toBytesBE();
+    const long = Long.fromNumber(value, false);
+    arr = new Uint8Array(long.toBytesBE()).buffer;
```

Comment:
The implicit conversion from `number[]` to `ArrayBuffer` was being rejected by the build phase ( [failed action](https://github.com/HathorNetwork/hathor-wallet-lib/actions/runs/9473676991/job/26101761010) )

An explicit conversion was implemented through `Uint8Array`, and according to [the unit tests](https://github.com/HathorNetwork/hathor-wallet-lib/blob/22459c067467a1599018ba3b60384622eabe42db/src/utils/buffer.ts#L57-L58) is working correctly.